File: app/src/main/java/com/karam/db/pojo/JobPOJO.java
package com.karam.db.pojo;

/**
 * POJO class for storing job attributes. This class will be used while interacting with the database.
 * Fill in the licence information.
 *
 * @author Rishabh Pandita
 */
public class JobPOJO {
    private long jobId;

    private long labId;

    private long contractorId;

    private boolean activeInd;

    /**
     * Returns jobId.
     *
     * @return Gets the value of jobId and returns jobId
     */
    public long getJobId() {
        return jobId;
    }

    /**
     * Sets the jobId
     */
    public void setJobId(long jobId) {
        this.jobId = jobId;
    }

    /**
     * Returns labId.
     *
     * @return Gets the value of labId and returns labId
     */
    public long getLabId() {
        return labId;
    }

    /**
     * Sets the labId
     */
    public void setLabId(long labId) {
        this.labId = labId;
    }

    /**
     * Returns contractorId.
     *
     * @return Gets the value of contractorId and returns contractorId
     */
    public long getContractorId() {
        return contractorId;
    }

    /**
     * Sets the contractorId
     */
    public void setContractorId(long contractorId) {
        this.contractorId = contractorId;
    }

    /**
     * Returns activeInd.
     *
     * @return Gets the value of activeInd and returns activeInd
     */
    public boolean isActiveInd() {
        return activeInd;
    }

    /**
     * Sets the activeInd
     */
    public void setActiveInd(boolean activeInd) {
        this.activeInd = activeInd;
    }
}

File: backend/POJO/JobPOJO.py
class JobPOJO:

    def setJobId(self, jobId):
        self.jobId = jobId
        return self

    def getJobId(self):
        return self.jobId

    def setLaborerId(self, laborerId):
        self.laborerId = laborerId
        return self

    def getLaborerId(self):
        return self.laborerId

    def setContractorId(self, contractorId):
        self.contractorId = contractorId
        return self

    def getContractorId(self):
        return self.contractorId

    def setActiveInd(self, activeInd):
        self.activeInd = activeInd
        return self

    def getActiveInd(self):
        return self.activeInd

    def setUpdateDtTm(self, updateDtTm):
        self.updateDtTm = updateDtTm
        return updateDtTm

    def getUpdateDtTm(self):
        return self.updateDtTm
